config/7.11.0/common.yaml

```diff
@@ -829,6 +829,40 @@ servers:
                       value: "[[.Jvm.GcContainerOptions]]"
                     #[[end]]
                     ## Jvm config END
+                    ## AMQ Streams BEGIN
+                    #[[if .KafkaExt.ExtDisabled]]
+                    - name: org.kie.kafka.server.ext.disabled
+                      value: "[[.KafkaExt.ExtDisabled]]"
+                    #[[end]]
+                    #[[if .KafkaExt.Topics]]
+                    - name: org.kie.server.jbpm-kafka.ext.topics
+                      value: "[[.KafkaExt.Topics]]"
+                    #[[end]]
+                    #[[if .KafkaExt.BootstrapServers]]
+                    - name: org.kie.server.jbpm-kafka.ext.bootstrap.servers
+                      value: "[[.KafkaExt.BootstrapServers]]"
+                    #[[end]]
+                    #[[if .KafkaExt.ClientID]]
+                    - name: org.kie.server.jbpm-kafka.ext.client.id
+                      value: "[[.KafkaExt.ClientID]]"
+                    #[[end]]
+                    #[[if .KafkaExt.AutocreateTopics]]
+                    - name: org.kie.server.jbpm-kafka.ext.allow.auto.create.topics
+                      value: "[[.KafkaExt.AutocreateTopics]]"
+                    #[[end]]
+                    #[[if .KafkaExt.GroupID]]
+                    - name: org.kie.server.jbpm-kafka.ext.group.id
+                      value: "[[.KafkaExt.GroupID]]"
+                    #[[end]]
+                    #[[if .KafkaExt.Acks]]
+                    - name: org.kie.server.jbpm-kafka.ext.acks
+                      value: "[[.KafkaExt.Acks]]"
+                    #[[end]]
+                    #[[if .KafkaExt.MaxBlockMs]]
+                    - name: org.kie.server.jbpm-kafka.ext.max.block.ms
+                      value: "[[.KafkaExt.MaxBlockMs]]"
+                    #[[end]]
+                    ## AMQ Streams END
```

Comment:
Env vars like these are not exists on Kie Server image, this configuration will not be passed correctly to the Kie Server image. 
On Kie Server image must be created env vars, witch can configured needed system properties. Then in Kie Operator we can pass needed values to the correct Kie Server's env vars.

Reply:
I was about to ask for the kieserver image changes.